Pythonwin/Scintilla/License.txt

```diff
@@ -2,19 +2,19 @@ License for Scintilla and SciTE
 
 Copyright 1998-2003 by Neil Hodgson <neilh@scintilla.org>
 
-All Rights Reserved 
+All Rights Reserved
```

Comment:
Thanks for the scintilla PR, but It would help me alot if you also split this up - just keep all Scintilla stuff out of this PR for now, and we can do that in a separate PR.